corehq/apps/repeaters/tests/test_repeater.py

```diff
@@ -105,10 +105,16 @@ def tearDown(self):
         self.case_repeater.delete()
         self.form_repeater.delete()
         FormProcessorTestUtils.delete_all_xforms(self.domain)
-        repeat_records = RepeatRecord.all()
-        for repeat_record in repeat_records:
-            repeat_record.delete()
-
+        results = RepeatRecord.get_db().view('receiverwrapper/repeat_records_by_next_check', reduce=False).all()
```

Comment:
nit: might be better to move to a dbaccessors file and marked `@unittesting_only`